virttest/utils_libvirtd.py

```diff
@@ -46,9 +46,9 @@ def __init__(self, session=None):
                             "utils_libvirtd module will not function normally")
         self.libvirtd = service.Factory.create_service(LIBVIRTD, run=runner)
 
-    def _wait_for_start(self, timeout=60):
+    def _wait_for_start(self, timeout=360):
```

Comment:
Good, but can we have it controlled using user param?, will benefit incase to adjust in future..

Reply:
I agree with @sathnaga I think we should generally avoid passing any hard coded values but use either config params or global (class) fields instead so the test consumer can decide to tweak the values consistently; as an example may serve https://github.com/avocado-framework/avocado-vt/blob/a75febd923b8d6c8e6720bef12ecd9874ff7fe00/virttest/virt_vm.py#L576

Reply:
Well I'm not a big fan of such customization especially on such low-level thing. How about waiting for someone actually having the intention to tweak it so they could suggest their favourite way? If not than I could turn it into a class variable but please consider the future user action first...

Reply:
(I mean it's generous enough while not blocking in case of a successful path)